frontend/src/Component/Share/Navbar/Sidebar.jsx

```
@@ -1,56 +1,100 @@
                     <>
                         <SidebarLink to="/teacher/dashboard" icon={HomeIcon} text="Dashboard" active={location.pathname.startsWith("/teacher/dashboard")} />
                         <SidebarLink to="/teacher/manage-students" icon={UserGroupIcon} text="Manage Students" active={location.pathname.startsWith("/teacher/manage-students")} />
                         <SidebarLink to="/teacher/grade-assignments" icon={ClipboardDocumentIcon} text="Grade Assignments" active={location.pathname.startsWith("/teacher/grade-assignments")} />
+
+                        {/* Teacher Courses with Sub-Options */}
+                        <button
+                            className={`w-full flex items-center justify-between py-2 px-3 rounded-md transition ${
+                                teacherCoursesOpen ? "bg-gray-300 text-gray-800 font-semibold" : "text-gray-300 hover:bg-gray-800"
+                            }`}
+                            onClick={() => setTeacherCoursesOpen(!teacherCoursesOpen)}
+                        >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)